server.js

```diff
@@ -52,20 +55,33 @@ app.use(bodyParser.urlencoded({
   extended: true
 }));
 
-// send assetPath to all views
+// Support for session
+app.use(session({
+  secret: "prototype-kit",
+  resave: false,
+  saveUninitialized: false
+}));
+
 app.use(function (req, res, next) {
+
+  // send assetPath to all views
   res.locals.asset_path="/public/";
-  next();
-});
 
-// Add variables that are available in all views
-app.use(function (req, res, next) {
+  // Add variables that are available in all views
   res.locals.serviceName=config.serviceName;
   res.locals.cookieText=config.cookieText;
   res.locals.releaseVersion="v" + releaseVersion;
+
   next();
+
 });
 
```

Comment:
I thought about it but I thought the code was self documenting? It automatically stores data?